### PR TITLE
Adding Spencer and more speaker info

### PR DESCRIPTION
**Pull request by cobusbernard**
Adding in Spencer Krum as a speaker and implemented the speaker bios.

### CODE REVIEW

File: content/events/2016-capetown/program.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ type = "event"
         <time>09:55-10:30</time>
       </div>
       <div class = "col-md-6 box">
-        Talk #2
+        <a href="/events/2016-capetown/program/daniel-maher">Spencer Krym (IBM) </br> Code Review for Operations</a>
```
Comment (phrawzty) [inline content/events/2016-capetown/program.md]: Either the link target or the name (and title) of this link are incorrect.

**Comment by cobusbernard** [comment]
Thanks @phrawzty for picking that one up, fixed.

File: content/events/2016-capetown/program.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ type = "event"
         <time>09:55-10:30</time>
       </div>
       <div class = "col-md-6 box">
-        Talk #2
+        <a href="/events/2016-capetown/program/spencer-krum">Spencer Krum (IBM) </br> Code Review for Operations</a>
```
Comment (phrawzty) [inline content/events/2016-capetown/program.md]: Whitespace around the `<br>`?

File: content/events/2016-capetown/program/seth-vargo.md [hunk 3]
```diff
@@ -8,9 +8,13 @@ aliases = ["/events/2016-capetown/program/seth-vargo/"]
...
-    <p><strong>Description:</strong></p>
-  </div>
+<div class="col-12">
+  <p><strong>Title:</strong>
+Building Secure Applications with Vault
+</p>
+
+<p><strong>Abstract:</strong></p>
+
+<p>Vault is a free and open source tool that disrupted the security software industry since its announcement in 2015. Due to its innovative architecture, Vault is highly scalable, incredibly flexible, operationally secure, and designed to fit almost any organization. In addition to behaving like an encrypted key-value store (like encrypted Redis ), Vault's dynamic secret backends are capable of generating database login credentials on demand, Amazon IAM users, and even act as a full certificate authority! Vault handles the leasing, key revocation, key rolling auditing, and more. Vault is truly a pleasure to work with as both an operator maintaining the Vault and a developer interacting with it. This live-demo will provide a 360-view of Vault, following the story of an evolving startup and their secret management solution.</p>
```
Comment (phrawzty) [inline content/events/2016-capetown/program/seth-vargo.md]: Whitespace here:
```encrypted Redis )
                ^```

**Pull request merged**